src/Component/Header/Header.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from "react";
+import "./style.scss";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Container } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import {
+  faSearch,
+  faUser,
+  faCartArrowDown,
+  faGlobe,
+  faBars,
+  faTimes,
+} from "@fortawesome/free-solid-svg-icons";
+function Header() {
+  const [toggleMenu, setToggleMenu] = useState(false);
+  const { t, i18n } = useTranslation();
+  const [language, setLanguage] = useState("en");
+
+  function handleChangeLanguage() {
```

Comment:
```suggestion
  function handleChangeLanguage(lang) {
```
Truyền tham số vào thay vì if else như này. Nếu > 2 ngôn ngữ thì lại phải else if nhiều lần